internal/config/config.go

```diff
@@ -89,6 +89,65 @@ type Config struct {
 
 // }
 
+// defaultConfig
+func defaultConfig() Config {
+	config := Config{}
+
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return applyEnvironmentConfiguration(config)
+	}
+
+	tcProp := filepath.Join(home, ".testcontainers.properties")
+	// Init from a file, ignore if it doesn't exist, which is the case for most users.
+	// The properties library will return the default values for the struct.
+	props, err := properties.LoadFiles([]string{tcProp}, properties.UTF8, true)
+	if err != nil {
+		return applyEnvironmentConfiguration(config)
+	}
+
+	if err := props.Decode(&config); err != nil {
+		fmt.Printf("invalid testcontainers properties file, returning an empty Testcontainers configuration: %v\n", err)
```

Comment:
suggestion: return an error.

Reply:
Agreed, hiding errors is bad and replacing a config, even one with errors, is even worse as the user is unlikely to notice and will be confused that things are not behaving correctly.

Reply:
Resolved in f4783a7bbe02fc582b58b0080f45c7f9bf1b01e2